src/pages/help-centre/_search-results.tsx

```diff
@@ -82,21 +94,18 @@ export const SearchSuccess = ({ suggested_topics, max_length }) => (
         {suggested_topics.slice(0, max_length).map((article, idx) => (
             <ListWrapper key={idx}>
                 <ListNoBullets>
-                    <StyledLink to={convertToHash(article.category, article.label)}>
+                    <StyledLink
+                        to={convertToHash({ category: article.category }, { label: article.label })}
+                    >
                         {localize(article.title)}
                     </StyledLink>
                 </ListNoBullets>
             </ListWrapper>
         ))}
     </>
 )
-SearchSuccess.propTypes = {
-    max_length: PropTypes.number,
-    onClick: PropTypes.func,
-    suggested_topics: PropTypes.array,
-}
 
-export const SearchError = ({ search }) => (
+export const SearchError = ({ search }: SearchSuccessProps) => (
```

Comment:
Isn't it better to have another `Type` for `SearchErrorType`?

Reply:
Interesting note. Done